### PR TITLE
Updated repo and removed unnecessary items

### PR DESCRIPTION
**Pull request by LouisjanB**
Removed color output.
Added Python 3 syntax
stringify the url
removed unused imports

### CODE REVIEW

File: validator.py [hunk 1]
```diff
@@ -1,22 +1,16 @@
 import sys
 from os import path, getcwd
 
-from colors import color
```
Comment (renatomefi) [inline validator.py:4]: What happened to the colors? :p

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
 openapi-spec-validator
```
Comment (renatomefi) [inline requirements.txt]: Can we lock the version and enable dependabot for future upgrades?

**Review by renatomefi — CHANGES_REQUESTED** [review]
haha nice of you touching a 4 year old abandoned track.

I think we have to fix the CI: GitHub Actions for opensource and gitlab for closed

File: validator.py (hunk 1 above)
Comment (LouisjanB) [reply validator.py:4]: You don't see the colours then this is running in the depths of testing. The idea is to get a result, true or false.

File: requirements.txt (hunk 2 above)
Comment (LouisjanB) [reply requirements.txt]: Yes....will add that in. Good idea

**Pull request closed without merging**